src/components/common/Footer.tsx

```
@@ -1,176 +1,244 @@
             </form>
             <p>
-              © {new Date().getFullYear()} TokenRadar Labs. All rights reserved. Forging next-generation open-source web3 tools.
+              © {new Date().getFullYear()} TokenRadar Labs. All rights
+              reserved. Forging next-generation open-source web3 tools.
             </p>
           </div>
         </div>
         <script
-          type="application/ld+json"
+          type='application/ld+json'
           dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)